app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/ListKnowledgeBaseItems.php

```diff
@@ -79,26 +81,37 @@ public function table(Table $table): Table
                     ->translateLabel()
                     ->searchable()
                     ->sortable(),
+                TextColumn::make('category.name')
+                    ->label('Category')
+                    ->translateLabel()
+                    ->toggleable()
+                    ->sortable(),
                 TextColumn::make('quality.name')
                     ->label('Quality')
                     ->translateLabel()
+                    ->toggleable()
                     ->sortable(),
                 TextColumn::make('status.name')
                     ->label('Status')
                     ->translateLabel()
+                    ->toggleable()
                     ->sortable(),
                 TextColumn::make('public')
                     ->label('Public')
                     ->translateLabel()
+                    ->toggleable()
                     ->sortable()
                     ->formatStateUsing(fn (bool $state): string => $state ? 'Yes' : 'No'),
-                TextColumn::make('category.name')
-                    ->label('Category')
-                    ->translateLabel()
-                    ->sortable(),
-                TextColumn::make('tags.name')
-                    ->badge()
-                    ->visible(fn (): bool => Feature::active('tags')),
```

Comment:
This column was removed here, but in the acceptance criteria it is still present, just modified and moved to be the last item in the list.

Reply:
Hi @Orrison ,

Tags (not default/not in filament column selector/not sortable)

But in Acceptance criteria it's mentioned as above. So by understanding that point isn't column should be removed from the list?

Reply:
Ah, that is a good point. Let me get clarity as to whether or not it should be removed.